integration-tests/pom.xml

```diff
@@ -206,6 +206,45 @@
     </profile>
 
     <profile>
+      <id>ossTest</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.tachyonproject</groupId>
+          <artifactId>tachyon-underfs-oss</artifactId>
+          <version>${project.version}</version>
+          <scope>test</scope>
+        </dependency>
+        <dependency>
+          <groupId>org.apache.httpcomponents</groupId>
+          <artifactId>httpcore</artifactId>
+          <version>4.4</version>
+          <scope>test</scope>
+        </dependency>
+      </dependencies>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <systemPropertyVariables>
+                <ufs>tachyon.underfs.oss.OSSUnderStorageCluster</ufs>
+                <!-- Change "myAccessId" to your oss access id for this test -->
+                <fs.oss.accessKeyId>myAccessId</fs.oss.accessKeyId>
+                <!-- Change "mySecretKey" to your oss access secret key for this test -->
+                <fs.oss.accessKeySecret>mySecretKey</fs.oss.accessKeySecret>
+                <!-- Change this endpoint for your oss service address for this test -->
+                <fs.oss.endpoint>http://oss-cn-hangzhou.aliyuncs.com</fs.oss.endpoint>
+                <!-- Change "your-bucket" to your oss bucket for this test -->
+                <ossBucket>oss://your-bucket/tachyon-integration-test/</ossBucket>
```

Comment:
NIT: For consistency, could you either change this to `my-bucket` or rename to `yourAccessId` and `yourSecretKey`?